README.md

```diff
@@ -9,136 +9,36 @@
+These SDKs allow us to create native iOS, tvOS, watchOS and macOS applications using the same UI controls we would in Objective-C and Xcode, except with the flexibility and elegance of a modern language (C#), the power of the .NET Base Class Library (BCL), and two first-class IDEs&mdash;Visual Studio for Mac and Visual Studio&mdash;at our fingertips.
 
-### Continuous Builds ###
+This repository is where we do development for the Xamarin.iOS and Xamarin.Mac SDKs. There are a few ways that you can contribute, for example:
 
-You can download continuous builds of our main development branches from [our wiki page](https://github.com/xamarin/xamarin-macios/wiki#continuous-builds).
+- [Submit bugs and feature requests](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions)
+- [Review source code changes](https://github.com/xamarin/xamarin-macios/pulls)
+- [Submit pull requests](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute#pull-requests) to resolve issues and fix bugs
 
-## Build requirements ##
+## Contributing
 
-* Autoconf, automake and libtool.
+If you are interested in fixing issues and contributing directly to the code base, please see the document [How to Contribute](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute), which covers the following:
 
-  You can use brew, or [this script](https://gist.github.com/rolfbjarne/3a979187ddd0855da073) to get
-  it directly from gnu.org (you'll have to edit your PATH to include /opt/bin if you use the script)
-	
-  To install brew and all the tool dependencies:
+- How to [build and run](https://github.com/xamarin/xamarin-macios/wiki/Build-&-Run) from source
+- The [development workflow](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute#work-branches), including [debugging](https://github.com/xamarin/xamarin-macios/wiki/Build-&-Run#debugging-applications-from-source) and [running tests](https://github.com/xamarin/xamarin-macios/blob/master/tests/README.md)
+- [Coding Guidelines](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute#coding-guidelines)
+- [Submitting pull requests](https://github.com/xamarin/xamarin-macios/wiki/How-to-Contribute#pull-requests)
 
-      $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-      $ brew update
-      $ brew install libtool autoconf automake bison flex
+## Feedback
 
-* CMake
+- Ask a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/xamarin.ios) or the [Xamarin Forums](https://forums.xamarin.com/)
+- [Request a new feature](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions#writing-good-bug-reports-and-feature-requests) on GitHub
+- [Vote](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions#before-submitting-an-issue) [on existing feature requests](https://github.com/xamarin/xamarin-macios/wiki/Submitting-Bugs-&-Suggestions#before-submitting-an-issue)
```

Comment:
"Vote" and "on existing feature requests" point both to the same link. Was that intentional, or are they supposed to point to different links?